src/rdfa-attributes.js

```diff
@@ -14,13 +14,14 @@ import { resolvePrefix } from './rdfa-helpers';
 */
 class RdfaAttributes {
 
-  constructor(domNode, knownPrefixes = defaultPrefixes) {
+  constructor(domNode, knownPrefixes = defaultPrefixes, options = {}) {
     if (domNode && domNode.getAttribute) {
       for (let key of rdfaKeywords) {
         this[`_${key}`] = domNode.getAttribute(key);
       };
       this.text = domNode.textContent;
-      this.setResolvedAttributes(knownPrefixes);
+
+      this.setResolvedAttributes(knownPrefixes, options.documentUrl);
```

Comment:
A minor improvement: setting `options.documentUrl` as a property on the RdfaAttributes object in the constructor prevents having to pass `documentUrl` as an additional arguments to all methods. The methods can just use `this.documentUrl` in that case.